docs/content/guides/recipes/howto-private-bicep-registry/index.md

```diff
@@ -0,0 +1,73 @@
+
+Before you get started, you'll need to make sure you have the following tools and resources:
+
+- [rad CLI]({{< ref "installation#step-1-install-the-rad-cli" >}})
+- [Bicep VSCode extension]({{< ref "installation#step-2-install-the-vs-code-extension" >}})
+- [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/)
+- [Radius initialized with `rad init`]({{< ref howto-environment >}})
+
+## Step 1: Setting up private OCI container registry authentication.
+Radius supports three authentication methods for accessing private container registries:
+- Basic Authentication: This method uses a username and password for authentication and is applicable to all OCI complaint registries. Obtain the username and password details used to login to private registry.
+- Azure Workload Identity: This federated identity-based authentication is used for connecting to Azure Container Registry (ACR). [Here]({{< ref howto-azure-provider-wi >}}) is the guide to setup Azure Workload Identity for Radius. Obtain `clientId` and `tenant ID` used during the setup.
+- AWS IRSA: This federated identity-based authentication is used for accessing Amazon Elastic Container Registry (ECR). [Here]({{< ref howto-aws-provider-irsa >}}) is the guide to setup the AWS IRSA for Radius. Obtain `roleARN` from the role created during the setup.
+
+## Step 2: Define a secret store resource
+
+Create a [Radius Secret Store]({{< ref "/guides/author-apps/secrets/overview" >}}) to manage and securely store the secerts information required for authenticating with a private registry. Define the namespace for the cluster that will contain your [Kubernetes Secret](https://kubernetes.io/docs/concepts/configuration/secret/) with the `resource` property. And specify the type of secret e.g. basicAuthentication,azureWorkloadIdeneity, awsIRSA. 
+
+> While this example shows a Radius-managed secret store where Radius creates the underlying secrets infrastructure, you can also bring your own existing secrets. Refer to the [secrets documentation]({{< ref "/guides/author-apps/secrets/overview" >}}) for more information.
+
+Secretstore example for secret type `awsIRSA`:
+{{< rad file="snippets/env.bicep" embed=true marker="//SECRETSTORE" >}}
+
+## Step 3: Configure authentication for private bicep registries and add a Bicep recipe.
+
+`recipeConfig` allows you to configure how Recipes should be setup and run. One available option is to specify the registry secrets for pulling Bicep recipes from private registries. For more information refer to the [Radius Environment schema]({{< ref environment-schema >}}) page.
+
+In your `env.bicep` file add an Environment resource, along with Recipe configuration which leverages the previously defined secret store for private OCI registry authentication. 
+
+{{< rad file="snippets/env.bicep" embed=true marker="//ENV" >}}
+
+
+## Step 5: Deploy your Radius Environment
+
+Deploy your new Radius Environment:
+
+```
+rad deploy ./env.bicep
+```
+
+## Done
```

Comment:
Instead of this "Done" header, I recommend showing the console output they should expect to see after deploying the environment, and then below the expected output the concluding "done" sentence can follow.

Reply:
updated it